### DISCUSSION TITLE
Welcome to rippled Discussions!

### DISCUSSION THREAD

**Discussion by RichardAH** [ripple/rippled#4228] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. If you have a technical issue that you already understand and you feel belongs in an _issue_ then please [open an issue](https://github.com/XRPLF/rippled/issues) instead. Otherwise please open a discussion. 
 
## Do ✅
 * Search existing discussions first before opening a new thread.
 * Ask questions you’re wondering about.
 * Share new ideas.
 * Engage with other community members.
 * Welcome others and remain open-minded.
 
## Don't 🚫
 * Advertise your project or business, or otherwise produce spam posts / threads.
 * Flame or abuse other users.
 * Post personal details.
 * Post anything that can't be found publicly already.

**Comment by intelliot** [comment]
Technical specifications (including drafts and discussions) should be posted under XRPL-Standards: https://github.com/XRPLF/XRPL-Standards/discussions
